### ISSUE TITLE
[BUG]Bundle size is quite large

### ISSUE THREAD

**Issue by ericp96** [microsoft/ApplicationInsights-JS#1234]
**Description/Screenshot**
![Screen Shot 2020-04-01 at 11 07 43 AM](https://user-images.githubusercontent.com/11936796/78165767-0ecc0200-7409-11ea-9a2d-bc9cdf0742eb.png)
Of the node modules I have, this one is far and away the largest.  I understand you guys are working hard maximizing the amount of telemetry you are gathering, but the expense of including this package seems to be too high.

**Steps to Reproduce**

 - OS/Browser: All
 - SDK Version [e.g. 22]: npm package 2.0.1
 - How you initialized the SDK:
```js
import { ApplicationInsights } from '@microsoft/applicationinsights-web';
let appInsightsRef = new ApplicationInsights({ config: { instrumentationKey } });
appInsightsRef.loadAppInsights();
appInsightsRef.setAuthenticatedUserContext(...);
appInsightsRef.trackPageView({ name, url: document.location.href });
appInsightsRef.trackPageViewPerformance({ name, url: document.location.href });
```

**Expected behavior**
Smaller size

**Additional context**
Webpack 4
Babel 7
Node 12
Tree shaking enabled
Split chunks enabled

**Comment by MSNev** [comment]
Hi,

Yes, we are aware of this and I have this work item / issue (#1076) as the initial work to reduce the overall size, once this work is complete I plan we will then re-evaluate how the SDK includes individual components.

I'm going to close this as a duplicate of #1076
